Custom babel plugin in .babelrc with parcel is not working.

# 🐛 bug report

## Description of the problem
[This](https://codesandbox.io/s/github/ismail-codar/fidan-jsx-examples) can be work on local machine but codesanbox throws error.

Your can use "Export to ZIP" from codesanbox-client and can be run with the following commands on your local computer.
```shell
npm install
npm start
```

## How has this issue affected you? What are you trying to accomplish?
Trying [fidanjs](https://github.com/ismail-codar/fidan) demos on codesanbox

### Link to sandbox: [link](https://codesandbox.io/s/github/ismail-codar/fidan-jsx-examples) (optional)

### Your Environment
Nodejs : v11.13.0

| Software         | Name/Version |
| ---------------- | ------------ |
| Сodesandbox      | latest
| Browser          | chrome latest
| Operating System | macos

also facing the same issue trying to add `styled-jsx/babel` to `.babelrc` plugins section using the Vanilla JS template with `parcel`. I get this error:

```
Error
Could not find/install babel plugin 'styled-jsx/babel': 
Cannot find module '@babel/plugin-styled-jsx/babel' from '/src'
```

Does `codesandbox` only look at the `src` folder for babel plugins?

Repro: https://codesandbox.io/s/intelligent-roentgen-wi6e2

--- a/packages/app/src/sandbox/eval/transpilers/babel/worker/babel-worker.js
+++ b/packages/app/src/sandbox/eval/transpilers/babel/worker/babel-worker.js
@@ -140,8 +140,8 @@ async function installPlugin(Babel, BFSRequire, plugin, currentPath, isV7) {
     console.warn(firstError);
 
     try {
-      /** Fast fallback - We assume that the user has the shortcode
-          styled-components = babel-plugin-styled-components
+      /** We assume that the user has the shortcode
+          react = @babel/react or styled-components = babel-plugin-styled-components
           and try to fetch the correct plugin for them
        */
 
@@ -160,7 +160,7 @@ async function installPlugin(Babel, BFSRequire, plugin, currentPath, isV7) {
       console.warn('Long path also failed ' + plugin + ' went wrong, got:');
       console.warn(secondError);
 
-      /** Slow fallback - If we still didn't get it, this means plugin wasn't downloaded
+      /** If we still didn't get it, this means plugin wasn't downloaded
           and that's why it could not be resolved.
 
           We can try to download it based on the first error
@@ -204,41 +204,17 @@ async function installPreset(Babel, BFSRequire, preset, currentPath, isV7) {
       Babel.availablePlugins,
       Babel.availablePresets
     );
-  } catch (firstError) {
-    try {
-      /** Fast fallback - We assume that the user has the shortcode
-          react = @babel/react and try to fetch the correct plugin for them
-       */
-      const prefixedName = getPrefixedPresetName(preset, isV7);
+  } catch (e) {
+    const prefixedName = getPrefixedPresetName(preset, isV7);
 
-      evaluatedPreset = evaluateFromPath(
-        fs,
-        BFSRequire,
-        prefixedName,
-        currentPath,
-        Babel.availablePlugins,
-        Babel.availablePresets
-      );
-
-      console.log('Second try succeeded');
-    } catch (secondError) {
-      console.warn('Long path also failed ' + preset + ' went wrong, got:');
-      console.warn(secondError);
-
-      /** Slow fallback - If we still didn't get it, this means preset wasn't downloaded
-          and that's why it could not be resolved.
-
-          We can try to download it based on the first error
-      */
-
-      console.warn('Downloading ' + preset);
-
-      evaluatedPreset = await downloadFromError(firstError).then(() => {
-        console.warn('Downloaded ' + preset);
-        resetCache();
-        return installPreset(Babel, BFSRequire, preset, currentPath, isV7);
-      });
-    }
+    evaluatedPreset = evaluateFromPath(
+      fs,
+      BFSRequire,
+      prefixedName,
+      currentPath,
+      Babel.availablePlugins,
+      Babel.availablePresets
+    );
   }
 
   if (!evaluatedPreset) {
@@ -249,8 +225,6 @@ async function installPreset(Babel, BFSRequire, preset, currentPath, isV7) {
     preset,
     evaluatedPreset.default ? evaluatedPreset.default : evaluatedPreset
   );
-
-  return evaluatedPreset;
 }
 
 function stripParams(regexp) {

--- a/packages/app/src/sandbox/eval/transpilers/babel/worker/babel-worker.js
+++ b/packages/app/src/sandbox/eval/transpilers/babel/worker/babel-worker.js
@@ -135,21 +135,45 @@ async function installPlugin(Babel, BFSRequire, plugin, currentPath, isV7) {
       Babel.availablePlugins,
       Babel.availablePresets
     );
-  } catch (e) {
+  } catch (firstError) {
     console.warn('First time compiling ' + plugin + ' went wrong, got:');
-    console.warn(e);
-    const prefixedName = getPrefixedPluginName(plugin, isV7);
+    console.warn(firstError);
 
-    evaluatedPlugin = evaluateFromPath(
-      fs,
-      BFSRequire,
-      prefixedName,
-      currentPath,
-      Babel.availablePlugins,
-      Babel.availablePresets
-    );
+    try {
+      /** We assume that the user has the shortcode
+          react = @babel/react or styled-components = babel-plugin-styled-components
+          and try to fetch the correct plugin for them
+       */
 
-    console.log('Second try succeeded');
+      const prefixedName = getPrefixedPluginName(plugin, isV7);
+      evaluatedPlugin = evaluateFromPath(
+        fs,
+        BFSRequire,
+        prefixedName,
+        currentPath,
+        Babel.availablePlugins,
+        Babel.availablePresets
+      );
+
+      console.log('Second try succeeded');
+    } catch (secondError) {
+      console.warn('Long path also failed ' + plugin + ' went wrong, got:');
+      console.warn(secondError);
+
+      /** If we still didn't get it, this means plugin wasn't downloaded
+          and that's why it could not be resolved.
+
+          We can try to download it based on the first error
+      */
+
+      console.warn('Downloading ' + plugin);
+
+      evaluatedPlugin = await downloadFromError(firstError).then(() => {
+        console.warn('Downloaded ' + plugin);
+        resetCache();
+        return installPlugin(Babel, BFSRequire, plugin, currentPath, isV7);
+      });
+    }
   }
 
   if (!evaluatedPlugin) {
@@ -160,6 +184,8 @@ async function installPlugin(Babel, BFSRequire, plugin, currentPath, isV7) {
     plugin,
     evaluatedPlugin.default ? evaluatedPlugin.default : evaluatedPlugin
   );
+
+  return evaluatedPlugin;
 }
 
 async function installPreset(Babel, BFSRequire, preset, currentPath, isV7) {

--- a/packages/app/src/sandbox/eval/transpilers/babel/worker/babel-worker.js
+++ b/packages/app/src/sandbox/eval/transpilers/babel/worker/babel-worker.js
@@ -140,8 +140,8 @@ async function installPlugin(Babel, BFSRequire, plugin, currentPath, isV7) {
     console.warn(firstError);
 
     try {
-      /** We assume that the user has the shortcode
-          react = @babel/react or styled-components = babel-plugin-styled-components
+      /** Fast fallback - We assume that the user has the shortcode
+          styled-components = babel-plugin-styled-components
           and try to fetch the correct plugin for them
        */
 
@@ -160,7 +160,7 @@ async function installPlugin(Babel, BFSRequire, plugin, currentPath, isV7) {
       console.warn('Long path also failed ' + plugin + ' went wrong, got:');
       console.warn(secondError);
 
-      /** If we still didn't get it, this means plugin wasn't downloaded
+      /** Slow fallback - If we still didn't get it, this means plugin wasn't downloaded
           and that's why it could not be resolved.
 
           We can try to download it based on the first error
@@ -204,17 +204,41 @@ async function installPreset(Babel, BFSRequire, preset, currentPath, isV7) {
       Babel.availablePlugins,
       Babel.availablePresets
     );
-  } catch (e) {
-    const prefixedName = getPrefixedPresetName(preset, isV7);
+  } catch (firstError) {
+    try {
+      /** Fast fallback - We assume that the user has the shortcode
+          react = @babel/react and try to fetch the correct plugin for them
+       */
+      const prefixedName = getPrefixedPresetName(preset, isV7);
 
-    evaluatedPreset = evaluateFromPath(
-      fs,
-      BFSRequire,
-      prefixedName,
-      currentPath,
-      Babel.availablePlugins,
-      Babel.availablePresets
-    );
+      evaluatedPreset = evaluateFromPath(
+        fs,
+        BFSRequire,
+        prefixedName,
+        currentPath,
+        Babel.availablePlugins,
+        Babel.availablePresets
+      );
+
+      console.log('Second try succeeded');
+    } catch (secondError) {
+      console.warn('Long path also failed ' + preset + ' went wrong, got:');
+      console.warn(secondError);
+
+      /** Slow fallback - If we still didn't get it, this means preset wasn't downloaded
+          and that's why it could not be resolved.
+
+          We can try to download it based on the first error
+      */
+
+      console.warn('Downloading ' + preset);
+
+      evaluatedPreset = await downloadFromError(firstError).then(() => {
+        console.warn('Downloaded ' + preset);
+        resetCache();
+        return installPreset(Babel, BFSRequire, preset, currentPath, isV7);
+      });
+    }
   }
 
   if (!evaluatedPreset) {
@@ -225,6 +249,8 @@ async function installPreset(Babel, BFSRequire, preset, currentPath, isV7) {
     preset,
     evaluatedPreset.default ? evaluatedPreset.default : evaluatedPreset
   );
+
+  return evaluatedPreset;
 }
 
 function stripParams(regexp) {

